packages/fastify-renderer/test/routes.spec.ts

```diff
@@ -138,7 +139,7 @@ describe('FastifyRenderer', () => {
       headers: { Accept: 'text/html' },
     })
 
-    expect(callOrder).toEqual(['transforms', 'heads', 'postRenders'])
+    expect(callOrder).toEqual(['heads', 'postRenders'])
```

Comment:
Why still have hooks that run in the parent process? The kind of hook that we use for css-in-js looks like this:

```typescript
hooks: [
      () => {
        const engine = new Server({ prefix: "g-" });

        return {
          name: "styletron",
          transform(app: ReactElement) {
            return React.createElement(StyletronProvider, { value: engine } as any, app);
          },
          postRenderHeads() {
            return engine.getStylesheetsHtml();
          },
        };
      },
```

and the hook that we use for Helmet looks like this:

```typescript
 return {
          name: "helmet",
          heads() {
            const helmet = Helmet.renderStatic();
            return [helmet.meta.toString(), helmet.link.toString()].join("\n");
          },
        };
```

Both of those need to run the heads or postrenders in the same context where the react rendering happened, as they use globals or context or what have you to track stuff that happened during render. Can we just move all the hooks into the inner worker? Would also scale a bit better.

Reply:
Ah, I didn't realize that this was the goal of the hooks.

Looks like instead of just importing the transform hook path, the whole hook needs to be imported. 

Nothing too difficult.